packages/react-native/ReactAndroid/src/main/java/com/facebook/react/bridge/DynamicFromObject.kt

```diff
@@ -0,0 +1,74 @@
+package com.facebook.react.bridge
+
+import com.facebook.common.logging.FLog
+import com.facebook.react.common.ReactConstants
+
+public class DynamicFromObject(private var mObject: Any?) : Dynamic {
```

Comment:
Can you rename the `mObject` to something else to avoid using Hungarian notation?

Because `object` would be a reserved word, I can propose as alternative:

```suggestion
public class DynamicFromObject(private var value: Any?) : Dynamic {
```

Reply:
Done